index.js

```diff
@@ -27,8 +27,9 @@ async function run() {
     }
 
     const pkgManager = (await ioUtil.exists("./yarn.lock")) ? "yarn" : "npm";
+    const installCmd = pkgManager === 'yarn' ? 'install --frozen-lockfile' : 'ci'
```

Comment:
Could you switch these over to double quotes to stay consistent with the line before it?

Reply:
Done